.github/workflows/issue-metrics.yml

```diff
@@ -8,35 +8,35 @@ on:
   pull_request:
     types: [opened]
     branches:
-      - main 
+      - main
```

Comment:
The oldest comment in the book: Rebase to remove these 😄

Reply:
Wait was it literally the oldest comments in the book? lol! Rebased ✅

Reply:
Wait were they literally the oldest comments in the codebase? lol! Rebased ✅